Code/Engine/Foundation/Strings/StringView.h

```diff
@@ -114,4 +114,14 @@ struct EZ_FOUNDATION_DLL ezStringView : public ezStringBase<ezStringView>
   const char* m_pEnd = nullptr;
 };
 
+namespace ezLiterals
```

Comment:
I'd prefer not to have a namespace for this, because it is inconvenient to write "using ..." everywhere to enable the feature. And we can't put "using ezLiterals" into our headers either, because then you have the name clash problem again. I think having a global "_ezsv" (or just "_ezs" ? ) would be better.

Reply:
We can put `using namespace ezLiterals` into our headers. `using namespace` statements can not only appear in the global scope, they can also appear within functions / methods. So for header files the using statement would be done inside the function / method that needs it.

As we are not using namespaces anywhere else in ez, I'm going to move to "_ezsv" instead. I don't want to use "_ezs" because I want to keep that for string literals (e.g. producing a `ezString`). To stay in line with STL which uses "sv" as suffix for string views I also want to stick with "_ezsv".